aws-rotate-keys/orb.yml

```diff
@@ -1,11 +1,11 @@
 version: 2.1
 description: An orb to rotate AWS keys
 
+orbs:
+  aws-cli: circleci/aws-cli@1
+
 executors:
-  default:
-    description: Default executor
-    docker:
-      - image: ovotech/aws-rotate-keys:latest
+  default: aws-cli/default
```

Comment:
Something to keep in mind - the `cimg/python:3.8-node` image doesn't provide the latest version of jq.

Reply:
cimg/python:3.7-node / cimg/python:3.8-node and all the next-gen CI images from circle are built on the same ubuntu base

https://github.com/CircleCI-Public/cimg-base#how-this-image-works

it provides the latest version of jq in from the Ubuntu repository

I have tested and the image works :)

Reply:
yeah, all I'm saying is that the 18.04 LTS doesn't ship with the latest version of jq, but I don't think this will break anything in our case